src/arch/cortex-m0/syscalls.S

```diff
@@ -13,12 +13,25 @@
 .thumb_func
```

Comment:
These changes are now redundant. The kernel is responsible for saving registers r4-r11. The version of this file `master` is correct.